.github/workflows/otbr.yml

```diff
@@ -233,6 +233,7 @@ jobs:
         retention-days: 1
 
   upload-coverage:
+    if: github.event.pull_request
```

Comment:
I think we want to upload coverage on pushes to main branch. Can we instead limit to the specific github repo?

```
github.repository == 'openthread/openthread'
```

Reply:
Updated in new push to use this condition.